python/tvm/intrin.py

```diff
@@ -488,6 +488,43 @@ def _rule_float_direct(op):
         return call_pure_extern(op.dtype, op.name, *op.args)
     return None
 
+@_register_func("tvm.default_trace_action")
+def tvm_default_trace_action(*args):
```

Comment:
suggest to name it `_tvm_default_trace_action`

Reply:
thanks for review, updated.